lib/callAll.js

```diff
@@ -328,7 +328,8 @@ module.exports = function(self, options) {
     // Also adds a class on <body> for both workflow modes
     if (req.user) {
       var workflowMode = req.session.workflowMode;
-      if (workflowMode === 'live') {
+      req.data.isPreview = req.session.isPreview;
+      if (workflowMode === 'live' || req.session.isPreview) {
         req.disableEditing = true;
```

Comment:
This is where we disable the apos-ui editing controls, Currently we only check for the live mode to disable the controls, I have also checked the "isPreview" flag(which is set if the preview option has been chosen by the user) to disable the the controls.